test/resizing.html

```diff
@@ -58,6 +58,16 @@
     </template>
   </test-fixture>
 
+  <test-fixture id="height-by-rows">
+      <template>
```

Comment:
nit: incorrect indentation here and below

Reply:
Fixed.